sdk/search/search-documents/CHANGELOG.md

```diff
@@ -9,13 +9,17 @@
 ### Breaking Changes
 
 - Update `SearchIndexerDataNoneIdentity.odatatype`. [#23495](https://github.com/Azure/azure-sdk-for-js/pull/23495)
+- Fix `SearchClient.getDocument` with `selectedFields` option returning a wide result with `undefined` non-optional fields [#23627](https://github.com/Azure/azure-sdk-for-js/pull/23627)
```

Comment:
This message is a bit confusing to me. Could you explain this in terms of its use case in addition to how the API shapes returned will be different. Also could you mention the addition of the type parameter that we discussed in a previous review that necessitates a breaking change?

Reply:
Done!